modules/logger/src/logger.c

```diff
@@ -45,7 +45,6 @@ static int addJSONString(FILE* fout, const char* jsonString)
         }
         else
         {
-            LogInfo("Message Received to Log: %s", jsonString);
```

Comment:
Will this line start printing out in all our samples now, and do we really want that behavior?